examples/java_test_suite_multiple_jdks/src/test/java/com/example/VersionTest.java

```diff
@@ -0,0 +1,15 @@
+package com.example;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class VersionTest {
+
+  @Test
+  public void test() {
+    List<String> properties = List.of("java.specification.version", "java.vm.vendor", "java.home");
+    for (String property : properties) {
+      System.out.printf("%s: %s\n", property, System.getProperty(property));
```

Comment:
Could we assert that the version is either 17 or 21?

Reply:
good idea, done!